app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/api.controllers");
+
+const app = express();
+
+app.get("/api/topics", getAllTopics);
+
+app.all("*", (req, res) => {
+  res.status(404).send({ msg: "Route not found" });
+});
+
+app.use((err, req, res, next) => {
```

Comment:
If you can't test for error throwing behaviour for any given task, I would not add boiler-plate error handling code right now. So probably best to remove until you are able to test these when testing for correct error handling for future endpoints!